ui/app/appointments/appointmentConfigInitialization.js

```diff
@@ -42,10 +42,12 @@ angular.module('bahmni.appointments').factory('appointmentConfigInitialization',
                 };
 
                 var getAllProviders = function () {
-                    var params = {v: "custom:(display,person,uuid)"};
+                    var params = {v: "custom:(display,person,uuid,attributes:(attributeType:(display),value,voided))"};
```

Comment:
Instead of filtering on client side, can we fetch the providers based on a certain attribute? I mean do we have a Web API which already does that? If yes can we use it.